lib/puppet/util/network_device/philips_hue/device.rb

```diff
@@ -1,52 +1,12 @@
-require 'faraday'
-require 'uri'
-require 'hocon'
-require 'hocon/config_syntax'
 require 'puppet/util/network_device/base'
-require 'json'
+require 'puppet/resource_api/transport/wrapper'
+# force registering the transport
+require 'puppet/transport/schema/philips_hue'
```

Comment:
this is done by the super, so doesnt need loaded here.

Reply:
Removed